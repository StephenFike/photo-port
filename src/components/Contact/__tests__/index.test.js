import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ContactForm from '..';

afterEach(cleanup);

describe('Contact component', () => {
    it('renders', () => {
        render(<ContactForm />)
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<ContactForm />)

        expect(asFragment).toMatchSnapshot();
    });

    it('validates Contact me text content', () => {
        const { getByTestId } = render(<ContactForm />)

        expect(getByTestId('text')).toHaveTextContent('Contact me')
    });

    it('validates submit button text content', () => {
        const { getByTestId } = render(<ContactForm />);

        expect(getByTestId('submit-btn')).toHaveTextContent('Submit')
    });
})