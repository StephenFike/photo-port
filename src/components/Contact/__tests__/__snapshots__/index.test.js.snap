// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component matches snapshot 1`] = `[Function]`;
